nx-X11/programs/Xserver/hw/nxagent/NXdispatch.c

```diff
@@ -1,9 +1,3 @@
-#ifdef NXAGENT_UPGRADE
```

Comment:
Ok, merging `nx-X11/programs/Xserver/hw/nxagent/NXdispatch.c` and `nx-X11/programs/Xserver/hw/nxagent/X/NXdispatch.c` sounds sane.